src/main/java/io/appform/dropwizard/sharding/dao/LookupDao.java

```diff
@@ -431,7 +431,7 @@ public ReadOnlyContext(
             this.getter = getter;
             this.entityPopulator = entityPopulator;
             this.key = key;
-            val skipFlag = System.getProperty("lookup.ro.skipTxn");
+            val skipFlag = System.getProperty("ro.skipTxn");
```

Comment:
This flag today is namespace agnostic. We should make it namespace aware. THis would help with cases where one container has two bundles and we want different configurations for each of them